src/status_im2/contexts/shell/jump_to/utils.cljs

```diff
@@ -126,7 +126,7 @@
 (defn change-shell-status-bar-style
   []
   (rf/dispatch [:change-shell-status-bar-style
-                (if (or (colors/dark?)
+                (if (or (quo.theme/get-theme)
```

Comment:
` (= :dark (quo.theme/get-theme))`?

btw, quo.theme also has `dark?` function

Reply:
whoops, yes this needs to be updated! good spot @Parveshdhull